db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql

```diff
@@ -1 +1,17 @@
 \echo 'Upgrade Queries not required for transition from $CURRENT_VERSION to $UPGRADE_VERSION'
```

Comment:
We can remove this statement as we have upgrade queries to run.

Reply:
removed as per the comment